modules/reference/pages/command-line-tools.adoc

```diff
@@ -695,6 +695,68 @@ Show help message and exit.
 
 
 
+[[spacewalk-repo-sync]]
+== Synchronize Software Repositories into {productname} Channels (spacewalk-repo-sync)
+
+[command]``spacewalk-repo-sync`` is an important tool on {productname}, which works most of the time in the background.
+With [command]``spacewalk-repo-sync`` you synchronize software repositories into {productname} channels.
+
+But for debugging or solving smaller issues it is useful to run it directly with special parameters.
+
+
+
+=== Normal Channel Synchronization
+
+Basic operation:
+
+[command]``spacewalk-repo-sync --list``::
+List all custom channels and the repositories assigned to them.
+[command]``spacewalk-repo-sync  --channel <custom-channel>``::
+Synchronize a single channel ``<custom-channel>`` to all repositories assigned to it with the {webui} or the API.
+
+
+
+=== Solve Checksum Problems
+
+Use the [option]``--deep-verify`` option to ignore cached package checksums, and thus solving checksum problems.
+
+
+
+=== Force Re-import Patches (Errata)
+
+Use the [option]``--force-all-errata`` to force re-importing all the patches (errata).
```

Comment:
```suggestion
Use the [option]``--force-all-errata`` option to force re-importing all the patches.
```
I feel like we would need to go into an exposition here about the use of the term "errata", I don't expect it's in common use outside of Red Hat.